mod_reforged/hooks/entity/tactical/player.nut

```diff
@@ -129,16 +134,26 @@
 	{
 		fillAttributeLevelUpValues(_amount, _maxOnly, _minOnly);
 
-		if (_amount == 0) return;
+		if (_amount < 2) return;
 		if (_maxOnly || _minOnly) return;	// Stars do not influence these level-ups
 
 		for (local i = 0; i != ::Const.Attributes.COUNT; i++)
 		{
 			if (this.m.Talents[i] == 2)
 			{
-				for (local j = 0; j < _amount; j++)
+				local indices = array(_amount);
+				foreach (j, _ in indices)
+				{
+					indices[j] = j;
+				}
+				local numIndicesToRandomize = ::Math.rand(2, _amount);
```

Comment:
This means that a brother can never have only +3 level-ups.
But on the flip-side a brother can have __no__ +3 level-ups.

You probably want to have some hardcoded fluff to have a small chance to roll all +3.